macaroon_service.go

```diff
@@ -170,40 +150,27 @@ func NewMacaroonService(cfg *MacaroonServiceConfig) (*MacaroonService, error) {
 	return &ms, nil
 }
 
-// Start starts the macaroon validation service, creates or unlocks the macaroon
-// database and, if we are not in stateless mode, creates the default macaroon
-// if it doesn't exist yet.
+// Start starts the macaroon validation service, creates or unlocks the
+// macaroon database and, if we are not in stateless mode, creates the default
+// macaroon if it doesn't exist yet.
 func (ms *MacaroonService) Start() error {
-	// Open the database.
-	var err error
-	ms.db, err = kvdb.GetBoltBackend(&kvdb.BoltBackendConfig{
```

Comment:
Can we extract this into a convenience function please? Otherwise we need to duplicate this code in all projects (loop/pool/faraday/litd).